pkg/subscriptions/subscriptions-view.jsx

```diff
@@ -0,0 +1,247 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "base1/cockpit",
+    "subscriptions/react",
+    "subscriptions/cockpit-components-listing",
+], function(cockpit, React, cockpitListing) {
+
+"use strict";
+
+var _ = cockpit.gettext;
+
+// Show details for an installed product
+var SubscriptionProductDetails = React.createClass({
+    render: function() {
+        return (
+            <div key={this.props.productId}>
+                <tr><td className="form-tr-ct-title">{_("Product name")}</td><td><span>{this.props.productName}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Product ID")}</td><td><span>{this.props.productId}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Version")}</td><td><span>{this.props.version}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Architecture")}</td><td><span>{this.props.arch}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Status")}</td><td><span>{this.props.status}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Starts")}</td><td><span>{this.props.starts}</span></td></tr>
+                <tr><td className="form-tr-ct-title">{_("Ends")}</td><td><span>{this.props.ends}</span></td></tr>
+            </div>
+        );
+    }
+});
+
+/* Implements a subset of the PatternFly Empty State pattern
+ * https://www.patternfly.org/patterns/empty-state/
+ * Special values for icon property:
+ *   - 'waiting' - display spinner
+ *   - 'error'   - display error icon
+ */
+var EmptyState = React.createClass({
```

Comment:
I would suggest calling this ```Curtains``` ... which is similar to but different in both use and look to ```EmptyState```.